go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/narqo/go-badge
 
-go 1.17
+go 1.20
```

Comment:
I believe, this directive declares the minimum compatible version of Go. Is there anything that blocks the project from working under 1.18, 1.19, and 1.20 without this change?